src/main/java/com/yelp/nrtsearch/server/luceneserver/search/SearchContext.java

```diff
@@ -49,6 +49,8 @@ public class SearchContext implements FieldFetchContext {
   private final Map<String, Object> extraContext;
   private final String queryNestedPath;
 
+  private final boolean isExplain;
```

Comment:
Nit: call the variable `explain` and it's getter `isExplain`

Reply:
kk, Explain test is here https://github.com/Yelp/nrtsearch/blob/master/src/test/java/com/yelp/nrtsearch/server/grpc/QueryTest.java#L838. It was added in the first explain pr. Let me know if you want me to add more

Reply:
@taoyyu would be good to have tests with different kinds of fetch (parallel by field, parallel by doc, single thread). Can do these one after other in the same test by changing the live settings.

Reply:
The parallel config is not live settings. It is the config file. I'll add the test in https://github.com/Yelp/nrtsearch/blob/master/src/test/java/com/yelp/nrtsearch/server/grpc/MultiSegmentTest.java with existing setup.